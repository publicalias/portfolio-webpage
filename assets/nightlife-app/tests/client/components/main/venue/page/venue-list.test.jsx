"use strict";

//local imports

const VenueList = require("../../../../../../scripts/client/components/main/venue/page/venue-list");

const { testWrapper } = require("../../../../test-helpers");

//global imports

const { initTestSnapshot } = require("redux/tests/client-tests");
const { reactTests } = require("redux/tests/react-tests");

//utilities

const { testShallow } = testWrapper(VenueList);

const testSnapshot = initTestSnapshot(testShallow);

//setup

beforeAll(reactTests.setup);
beforeEach(reactTests.inject(VenueList));

//venue list

test("VenueList should match snapshot", () => testSnapshot());
