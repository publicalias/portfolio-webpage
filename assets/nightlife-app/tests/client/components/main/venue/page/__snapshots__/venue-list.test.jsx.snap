// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`VenueList should match snapshot 1`] = `""`;
